docs/architecture/adr-050-sign-mode-textual-annex1.md

```diff
@@ -0,0 +1,89 @@
+# ADR 049: SIGN_MODE_TEXTUAL - Annex 1 Value Renderers
+
+## Changelog
+
+- Dec 06, 2021: Initial Draft
+
+## Status
+
+Draft
+
+## Abstract
+
+This Annex describes value renderers, which are used for displaying values in a human-friendly way.
+
+## Value Renderers
+
+Value Renderers describe how values of different Protobuf types should be automatically rendered. Value renderers can be formalized as a set of bijective functions `func renderT(value T) string`, where `T` is one of the below Protobuf types on which this spec is defined.
+
+### Protobuf `number`
+
+- Applies to numeric integer types (`uint64`, etc.) and their casted types (e.g. in Golang: `sdk.Dec`, `sdk.Int`).
+- Formatting with `'`s for every three integral digits.
+- Usage of `.` to denote the decimal delimitier.
+
+### Examples
+
+- sdk.Int, integers: `1000` -> `1'000`
+- sdk.Dec; `1000000.00` -> `1'000'000.00`
+
+### `coin`
+
+- Applies to `cosmos.base.v1beta1.Coin`.
+- Denoms are converted to `display` denoms using `Metadata` (if available). **This requires a state query**.
+- Amounts are converted to `display` denom amounts and rendered as `number`s above
```

Comment:
add clarity that we keep the value of state in chain. Add note on `atom` vs `ATOM`, and chain should decide what's shown to users